en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,87 @@
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the release policy for Silverstripe CMS. It applies to all [commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the silverstripe GitHub organisation that are not commercially supported are updated on a best effort basis.
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+The purpose of this policy is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
+
+## Release types and cadence
+
+Silverstripe CMS adheres to [Semantic versioning](https://semver.org/) for its release philosophy.
+
+### Major releases `(x.0.0)`
+
+Major releases include architectural updates, allow breaking changes and major dependency changes.
+
+- **Purpose:** To deliver breaking changes and architectural updates that cannot be shipped in minor releases due to backward compatibility constraints.
+
+- **Backward compatibility:** Major releases are **not backward compatible**.
+
+- **Frequency:** New Silverstripe CMS major versions are released every **two years**.
+
+- **Tagging period:** New major releases are typically tagged as stable between **April and June** of their release year.
+
+- **Scope:** This policy applies to all Silverstripe CMS commercially supported modules. At launch, the list of commercially supported modules for that specific major release line will be explicitly defined.
+
+- **Pre-release period:** A stable major release is always preceded by a **beta period of at least three months** to allow for extensive testing and feedback.
+
+- **Dependency alignment:** At launch, Silverstripe CMS major releases aim to:
+
+  - Support all PHP versions currently in full support.
+  - Have all PHP and JavaScript dependencies on their latest stable versions.
+
+- **Upgrade path:** Silverstripe CMS commits to providing a clear upgrade path to new major releases, including comprehensive documentation on all breaking changes and migration guides.
+
+### Minor releases `(x.y.0)`
+
+Minor releases provide ongoing value and improvements within a stable major release line. They are backwards compatible within the major release line and designed to be easy to upgrade to.
+
+- **Purpose:** To ship new features, enhancements, performance improvements, and bug fixes that cannot be shipped in a patch releae due to API additions.
+
+- **Backward compatibility:** Minor releases are **backward compatible** within their major release line. They will not introduce breaking changes.
+
+- **Frequency:** Minor releases are shipped every **six months** within an active major release line, typically targeted for **April and October**. There will be **five** minor releases per major release.
```

Comment:
```suggestion
- **Frequency:** Minor releases are shipped every **six months** within an active major release line around **April and October**.
```

Reply:
The existing policy currently says "Silverstripe CMS **_aims to ship_** a new minor releases every 6 months" (emphasis added).

This  "aims to" technically gives us an out if for some reason we want to skip a release or sneak an extra one in. We should probably get product owner approval before changing that language.